extensions/ql-vscode/src/skeleton-query-wizard.ts

```diff
@@ -141,7 +141,7 @@ export class SkeletonQueryWizard {
       maxStep: 3,
     });
 
-    return await askForLanguage(this.cliServer, false);
+    return await askForLanguage(this.cliServer, true);
```

Comment:
That `throwOnEmpty` argument is confusingly named, but that's not the fault of this PR 😅

Reply:
I agree! To me, it's something more like "always throw, but less aggressively" 🙃 

I'll leave it as-is for now though.